src/main/resources/assets/securitycraft/lang/pt_br.json

```diff
@@ -0,0 +1,1198 @@
+	"gui.securitycraft:scManual.recipe.keypad_smoker": "Coloque um fumante e clique com a direita com um painel -chave.",
+	"gui.securitycraft:scManual.recipe.reinforced": "Qualquer bloco reforçado pode ser criado da seguinte maneira, se não for especificado de outra forma neste livro.Coloque o equivalente a baunilha (por exemplo, pedra reforçada -> pedra) e quebre -a com um reforçador universal de blocos, ou clique com o reforço universal do bloco, coloque -o no slot e feche a GUI.",
+	"gui.securitycraft:scManual.recipe.reinforced_crystal_quartz": "Feito de blocos regulares de quartzo de cristal em um reforçador universal de bloco.",
+	"gui.securitycraft:scManual.recipe.reinforced_iron_trapdoor": "Qualquer bloco reforçado pode ser criado da seguinte maneira, se não for especificado de outra forma neste livro.Coloque o equivalente a baunilha (por exemplo, pedra reforçada -> pedra) e quebre -a com um reforçador universal de blocos, ou clique com o reforço universal do bloco, coloque -o no slot e feche a GUI.",
+	"gui.securitycraft:scManual.recipe.sc_manual": "O manual de segurança de segurança está disponível apenas através do inventário criativo ou digitando \"/sc help\" (sem cotações).",
+	"gui.securitycraft:scManual.reinforced": "Blocos reforçados",
+	"gui.securitycraft:scManual.block_mines": "Minas de bloco",
+	"gui.securitycraft:scManual.reinforced_buttons": "Botões reforçados",
+	"gui.securitycraft:scManual.reinforced_pressure_plates": "Placas de pressão reforçadas",
+	"gui.securitycraft:scManual.keycards": "keycards",
+	"gui.securitycraft:scManual.secret_signs": "Sinais secretos",
+	"gui.securitycraft:scManual.block_reinforcers": "Reforçadores de blocos universais",
+	"gui.securitycraft:scManual.designedBy": "Projetado por: %s",
+	"gui.securitycraft:scManual.patreon.title": "Nossos clientes:",
+	"gui.securitycraft:scManual.patreon.loading": "Carregando...",
+	"gui.securitycraft:scManual.patreon.error": "Erro buscando clientes!",
+	"gui.securitycraft:ims.target": "Alvo:",
+	"gui.securitycraft:invScan.emit_redstone": "Emite Redstone em itens encontrados: %s",
+	"gui.securitycraft:invScan.check_inv": "Itens encontrados na loja: %s",
+	"gui.securitycraft:invScan.yes": "Sim",
+	"gui.securitycraft:invScan.no": "Não",
+	"gui.securitycraft:invScan.prohibitedItems": "Itens proibídos",
+	"gui.securitycraft:invScan.mode.admin": "Modo de administrador",
+	"gui.securitycraft:invScan.mode.view": "Modo de visualização",
+	"gui.securitycraft:invScan.storage": "Armazenar",
+	"gui.securitycraft:invScan.notInstalled": "o %s não está instalado.",
+	"gui.securitycraft:protectedFurnace.name": "Forno protegido",
+	"gui.securitycraft:logger.logged": "Jogadores registrados:",
+	"gui.securitycraft:logger.date": "Encontro: %s",
+	"gui.securitycraft:mrat.notBound": "Não vinculado!",
+	"gui.securitycraft:mrat.mineLocations": "Meu em %s",
+	"gui.securitycraft:mrat": "Ferramenta de acesso remoto para minas",
+	"gui.securitycraft:mrat.defuse": "DOUSUSUSE",

```

Comment:
I think that word is incorrect (at least according to Google Translate) and it's in ALL CAPS.